bioconda_recipes_issue_responder/src/main.ts

```diff
@@ -65,7 +65,8 @@ async function fetchArtifacts(ID) {
   res = res.replace(/:pretty-path/g, "\"pretty-path\":");
   res = res.replace(/:url/g, "\"url\":");
   let artifacts = JSON.parse(res).filter(x => x['url'].endsWith(".tar.gz") || x['url'].endsWith(".tar.bz2") || x['url'].endsWith("/repodata.json")).map(x => x['url']);
-  return(artifacts);
+  let unique = (array) => [...new Set(array)];
```

Comment:
`Set`s retain the insertion order, acc. to my personal ECMAScript guru (aka brother), neat.